### ISSUE TITLE
how to generate run time code coverage

### ISSUE THREAD

**Issue by nagaraju-sunkara** [bmuschko/gradle-clover-plugin#124]

**Comment by nagaraju-sunkara** [comment]
Provide a gradle task to genearte instrument classes with coverage reports genearted at a regular interval

**Comment by nagaraju-sunkara** [comment]
Yes. I am looking for instrumented classes to have a run time coverage

**Comment by Alex-Vol-SV** [comment]
When building instrumented code your application must include clover.jar as well. Instrumented code has references to Clover classes so your application needs the JAR file.

See documentation of plugin here: [Properties](https://github.com/bmuschko/gradle-clover-plugin#project-properties)

**Comment by nagaraju-sunkara** [comment]
Thanks. I could able to generate instrumented JAR file. but it is missing the clover jar  as part of build thus throws exception "No class def found".

**Comment by Alex-Vol-SV** [comment]
You are responsible for adding this JAR to your application when testing with instrumented JAR file. That is expected of you. I am not going to embed Clover JAR contents in your application JAR file. You can do that if you wanted without changes to the plugin if you think it would help. But the normal way of using the instrumented JAR file is by adding the clover.jar in the application, either on the classpath or inside a WAR file if you are deploying in a container.

**Comment by nagaraju-sunkara** [comment]
i could able to add jar at runtime during build now. 

Can you give me the steps to have a singleCloverDatabase  when it executes the clover tasks?

**Comment by Alex-Vol-SV** [comment]
This feature is not implemented yet. We do not handle a clover database produced during an external invocation of instrumented code. If your instrumented JAR is used by an integration test harness managed by Gradle it might be possible to do such a merge of the databases.
